bundle/manifests/obo-prometheus-operator-admission-webhook_v1_service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
```

Comment:
nit: this line can be removed

Reply:
Certainly but this needs to be done upstream and kustomize is only pulling this service from the release.

Reply:
hmm, it's probably added by kustomize because it isn't present here:

https://raw.githubusercontent.com/rhobs/obo-prometheus-operator/v0.60.0-rhobs1/example/admission-webhook/service.yaml